package.json

```diff
@@ -59,7 +59,8 @@
     "reboot": "yarn clean && yarn reset && yarn install && yarn start",
     "rollup.bundle": "rollup -c -m",
     "rollup.watch": "yarn rollup -w",
-    "rollup": "yarn rollup.bundle"
+    "rollup": "yarn rollup.bundle",
+    "test": "tsc && wtr"
```

Comment:
If I haven't run storybook (and compiled the stylesheets), then the TypeScript compiler complains about the references to non-existing files.

```
src/components/base/outline-accordion/outline-accordion.ts:3:29 - error TS2307: Cannot find module './outline-accordion.css.lit' or its corresponding type declarations.

3 import componentStyles from './outline-accordion.css.lit';
```

Should we add `generate.component-css` to this task?

Reply:
@mabry1985 If we change the above `yarn test` command as described, I think we may be good to go here.